peachjam/templates/peachjam/_court_list.html

```diff
@@ -23,7 +23,7 @@ <h4 class="mb-2">{{ court_class.name }}</h4>
           {% if court_class.show_listing_page %}
             <div class="card-footer">
               {% block court_class_link %}
-                <a href="{% url 'court_class' court_class.slug %}">{% blocktrans with court=court_class %}Explore {{ court }}{% endblocktrans %} →</a>
```

Comment:
If you make this `court=court_class.name` then it should Just Work without the below.